plugins/c8y_configuration_plugin/src/main.rs

```diff
@@ -208,7 +209,7 @@ fn create_operation_files(config_dir: &str) -> Result<(), anyhow::Error> {
         &format!("{config_dir}/c8y/c8y-configuration-plugin.toml"),
         "root",
         "root",
-        0o644,
+        0o775,
```

Comment:
Why this change ? There is no point to have this file executable.

Reply:
Inotify needs this to watch the file content

Reply:
This is definitely not the case. To watch a file one just needs to be able to read it.

That said you might experience an issue if you edit the file using an editor which rename a temporary file instead of opening it for write. Since you are watching `WatchMask::CLOSE_WRITE on a specific file, those events will never be observed because the watched file is moved.

=> You have to watch the directory containing the configuration file.

Reply:
I see what you mean... I was not using inotify correctly